services/client/src/pages/events/components/events.js

```diff
@@ -14,7 +14,7 @@ class Events extends Component {
 
   componentDidMount() {
     const { fetchData } = this.props;
-    fetchData(`${process.env.REACT_APP_EVENTS_SERVICE_URL}/events`);
```

Comment:
This line will need to be reverted otherwise staging and production builds will be unable to load `events`.

Does the following work for you instead?
```
export REACT_APP_EVENTS_SERVICE_URL=http://localhost
```

Reply:
My bad! It was a temp fix I didn't mean to push. Amending in a commit now

Reply:
Also that does work! Thanks :)